app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Platform.scala

```diff
@@ -6,12 +6,71 @@ import java.sql.Timestamp
 import io.circe._
 import io.circe.generic.JsonCodec
 import io.circe.syntax._
+import io.circe.generic.semiauto._
+import cats.syntax.either._
+
 
-@JsonCodec
 case class Platform(
   id: UUID,
   name: String,
-  settings: Json,
+  publicSettings: Platform.PublicSettings,
   isActive: Boolean,
-  defaultOrganizationId: Option[UUID]
+  defaultOrganizationId: Option[UUID],
+  privateSettings: Platform.PrivateSettings
 )
+
+object Platform {
+
+  def tupled = (Platform.apply _).tupled
+
+  case class PublicSettings(
+    emailUser: String,
+    emailSmtpHost: String,
+    emailIngestNotification: Boolean,
+    emailAoiNotification: Boolean
+  )
+
+  case class PrivateSettings(emailPassword: String)
+
+  implicit val encodePublicSettings: Encoder[PublicSettings] = new Encoder[PublicSettings] {
```

Comment:
Here, where you're just doing "take all the fields and throw them into the json to encode, and take all the fields I need from the incoming json to decode" -- you can just use `@JsonCodec`.